packages/core/integration-tests/test/hmr.js

```diff
@@ -23,7 +24,7 @@ async function closeSocket(ws: typeof WebSocket) {
   ws.close();
   await new Promise(resolve => (ws.onclose = resolve));
 }
-
+// flowlint-next-line unclear-type:off
```

Comment:
This should be resolvable once we have typings for the ws package :)

Reply:
Do you suggest removing this?

Reply:
not now — when we add typings for ws :)